apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachment-document-creator/AttachmentCreateDocumentInput.vue

```diff
@@ -179,7 +179,11 @@ export default {
         doc.drive = this.currentDrive.title;
         doc.date = doc.created;
         this.$root.$emit('add-new-created-document', doc);
-        this.$root.$emit('end-loading-attachment-drawer');
+        document.dispatchEvent(new CustomEvent('entity-attachments-updated'));
```

Comment:
@GouadriaHanen could you remove the instruction from here ?

Reply:
done